lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -777,7 +777,17 @@ module Resources
             'start': '2012-05-17T15:33:18.000Z'
           }
         }
-      }
+      },
+      {
+        'path': '/name/0/id',
```

Comment:
just to be consistent with the rest of the examples on this resource, can we put the op first and then the path. on all your examples

Reply:
Changed in all.
https://github.com/cerner/fhir.cerner.com/commit/d6ef4e31556be3decb0db09531e1e82211b51d12#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR783